javascript/ql/src/semmle/javascript/security/dataflow/CodeInjectionCustomizations.qll

```diff
@@ -197,11 +197,63 @@ module CodeInjection {
   }
 
   /**
-   * A value interpreted as a tempalte by the `dot` library.
+   * A value interpreted as a template by the `handlebars` library.
```

Comment:
Nit: Unlike recent PRs, you no longer link to the npm URL. I prefer this URL-less style though.

Perhaps we should drop the npm URLs in the source code completely.